tron/config/config_utils.py

```diff
@@ -197,6 +198,19 @@ def validator(value, config_context):
     return validator
 
 
+def build_dict_value_validator(item_validator, allow_empty=False):
+    """Build a validator which validates values of a dict, and returns a dict"""
+
+    def validator(value, config_context):
+        msg = "Duplicate name %%s at %s" % config_context.path
```

Comment:
are the double-% on purpose here?

Reply:
Yeah, this was a literal copy/paste of the `build_dict_name_validator` -- this actually feels a bit unnecessary, given we're already assuming we're working with a dict which will prevent us from having duplicate keys.  I'll go ahead and try to return a standard dict instead of this.